pages/1.11/installing/troubleshooting/index.md

```diff
@@ -173,16 +176,18 @@ Publicly accessible applications are run in the public agent node. Public agent
 
 * You might not be able to SSH to agent nodes, depending on your cluster network configuration. We have made this a little bit easier with the DC/OS CLI. For more information, see [SSHing to a DC/OS cluster][6].
 
-* You can get the IP address of registered agent nodes from the **Nodes** tab in the [DC/OS web interface][7]. Nodes that have not registered are not shown.
+* You can get the IP address of registered agent nodes from the **Nodes** tab in the DC/OS Dashboard. Nodes that have not registered are not shown.
 
 * SSH to your agent node and enter this command to view the logs from boot time:
 
     ```bash
-    journalctl -u dcos-mesos-slave -b
+    journalctl -u dcos-marathon -b
```

Comment:
Good catch @judithpatudith. Yeah the `dcos-marathon` doesn't exist on the agents but only the masters. This will return `-- No entries --` on the node. We can revert this back to `journalctl -u dcos-mesos-slave -b` .